pkg/cmd/release-payload-controller/release_creation_job_controller.go

```diff
@@ -171,9 +153,34 @@ func (c *ReleaseCreationJobController) sync(ctx context.Context, key string) err
 		return err
 	}
 
+	// If the Coordinates are already set, then don't do anything...
+	if len(originalReleasePayload.Status.ReleaseCreationJobResult.Coordinates.Namespace) > 0 && len(originalReleasePayload.Status.ReleaseCreationJobResult.Coordinates.Name) > 0 {
+		return nil
+	}
+
+	klog.V(4).Infof("Syncing ReleaseCreationJobResult for ReleasePayload: %s/%s", originalReleasePayload.Namespace, originalReleasePayload.Name)
+
+	// Update the Coordinates of the ReleaseCreationJobResult
+	coordinates := originalReleasePayload.Status.ReleaseCreationJobResult.Coordinates.DeepCopy()
+	coordinates.Namespace = c.jobsNamespace
+	coordinates.Name = originalReleasePayload.Name
+
 	releasePayload := originalReleasePayload.DeepCopy()
+	releasePayload.Status.ReleaseCreationJobResult.Coordinates = *coordinates
```

Comment:
if you're setting this, set *all* the fields.  There should be no status and no message, so clear them to force another controller to rediscover and set them.